sessionctx/stmtctx/stmtctx.go

```diff
@@ -166,6 +166,9 @@ type StmtHints struct {
 	NoIndexMergeHint        bool
 	// EnableCascadesPlanner is use cascades planner for a single query only.
 	EnableCascadesPlanner bool
+	// ForceNthPlan indicates the countDown number for finding physical plan.
+	// -1 for disable.
```

Comment:
can we use 0 to disable since 0-th something is rarely used?

Reply:
It seems not, 0 may appear during the calculation, which means that the clock is cleared. (different with "the clock is disabled")